applications/ParticleMechanicsApplication/custom_constitutive/hyperelastic_3D_law.cpp

```diff
@@ -660,16 +660,15 @@ Matrix& HyperElastic3DLaw::Transform2DTo3D (Matrix& rMatrix)
 {
     if (rMatrix.size1() == 2 && rMatrix.size2() == 2)
     {
-        rMatrix.resize( 3, 3, true);
+        const BoundedMatrix<double,2,2> temp_matrix = rMatrix;
```

Comment:
you cannot resize a bounded matrix!

Reply:
=> resize happens in the next line
@pooyan-dadvand shouldn't this throw a compiler error?

Reply:
sorry I misread the code, nevermind